frontend/src/services/graphql/AirflowCluster/index.js

```diff
@@ -0,0 +1,6 @@
+export * from './createAirflowProject';
```

Comment:
Airflow is not used in the project and can be deleted. All these API calls will fail when they try to reach backend. The BE functionality had been deleted even before modularization. 
Part of the API calls were already deleted on FE, not sure how these 6 survived though

Reply:
Yeah it was weird! not quite sure though why many old files showed up in the merging process